orangecontrib/bioinformatics/widgets/OWMarkerGenes.py

```diff
@@ -370,6 +399,14 @@ def _filter_table(self):
         model = self.view.model()
         assert isinstance(model, SearchableTableModel)
         model.update_model(str(self.filter_text))
+        self.commit()
+        self.update_data_info()
+
+    def update_data_info(self):
+        filtered_data = len(self.view.model().source)
+        all_data = len(self.view.model()._data)
+        self.info.set_input_summary(f"Shown : {str(filtered_data)}/{str(all_data)}")
+        self.info.set_output_summary(f"Selected: {str(len(self.selected_genes))}")
```

Comment:
Those two lines are longer than 79 characters. As far I know we are still keeping the line lengths at maximum 79 as it is declared PEP8.